### PR TITLE
Component

### PR DESCRIPTION
**Pull request by josuerushanika**
**In this project,It is about development of the Math Magicians app.l develop a React component that will hold the core functionality of a calculator.**

**Project requirements**

_Create a new directory called components
Inside components, create a new Calculator.js file
Implement the design only (JSX and CSS_

### CODE REVIEW

File: src/Components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      <button type="button">2</button>
+      <button type="button">1</button>
+      <button type="button" className="operationsign"> + </button>
+    </div>
+
+    <div className="symbole">
+      <button type="button" className="zero">0</button>
+      <button type="button" className="dot">.</button>
+      <button type="button" className="operationsign">=</button>
+    </div>
```
Comment (andersonlebon) [inline src/Components/Calculator.js]: - [ ] Great work @josuerushanika 👏  but just one thing left. Kindly ensure your `AppCalculator` component has at least one child component as mentioned in the requirements.

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @josuerushanika 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## To highlight :champagne: 
- Your README file is professional :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/Calculator.js (hunk 1 above)
Comment (josuerushanika) [reply src/Components/Calculator.js]: Thanks it is done

**Review by pinkmoon25 — APPROVED** [review]
Hi @josuerushanika , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters :heavy_check_mark: 
Gitflow :heavy_check_mark: 
Readme :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by josuerushanika** [comment]
Thank You

**Pull request merged**